draft-ietf-cbor-7049bis.md

```diff
@@ -408,27 +415,33 @@ floats as integers or vice versa, perhaps to save encoded bytes.
 
 A CBOR data item  ({{cbor-data-models}}) is encoded to or decoded from
 a byte string as described in this section.  The encoding is
-summarized in {{jumptable}}.
+summarized in {{jumptable}}.  An encoder MUST produce only valid
```

Comment:
Note from the 2018-12-19 telecon: this cannot be a constraint on generic encoders, since they don't know what's valid for tags. I intended it to constrain protocol output, but that needs to go somewhere else.